plugins/content_distribution/providers/cross_kaltura/lib/batch/CrossKalturaDistributionEngine.php

```diff
@@ -934,6 +969,35 @@ protected function syncAssetsContent(KalturaServiceBase $targetClientService, $t
 		}
 	}
 
+	protected function syncTargetQuiz(KalturaServiceBase $targetClientService, $quizObject, $targetEntryId)
+	{
+		$quiz = null;
```

Comment:
you call similar code above so better have it as function that gets entry id

Reply:
Done